openoffice.txt

```diff
@@ -2,13 +2,16 @@
 Name = Apache OpenOffice
 Version = 4.1.10
 License = AL v2.0
+LicenseInfo = 1
 Description = Open source office suite.
 Category = 6
 URLSite = https://www.openoffice.org/
 URLDownload = https://download.sourceforge.net/project/openofficeorg.mirror/4.1.10/binaries/en-US/Apache_OpenOffice_4.1.10_Win_x86_install_en-US.exe
 SHA1 = 68d73772b0f9559a08d65ca556230fe32530af41
 CDPath = none
 SizeBytes = 142587736
+Icon = openoffice.ico
+Screenshot1 = https://home.apache.org/~mseidel/AOO-ReactOS.png
```

Comment:
Please use a well recognized imagehost (like imgur.com) instead of your own domain.

Reply:
OK, then I will remove it since I am not going to create an account at imgur.com.
I didn't know that apache.org is not well recognized? :-D

Reply:
In principle "we" should host the icons and screenshots (as these are part of the package database)

Reply:
You don't need an account at imgur to upload a picture. Also, Mark probably meant your own userspace (`~mseidel`) on this domain, rather the domain itself.

Reply:
OK, I will think about Imgur.
Would a screenshot from https://www.openoffice.org be better?

Reply:
The idea behind this is that other domains are not going to track our users.
Ideally we would have our own image host, but that would be yet another thing to maintain, so we chose for the simple middle-ground:
Using well-known image hosts.

Reply:
Done...